tests/jballerina-integration-test/src/test/resources/observability/metrics_tests/03_addTagToMetrics_function.bal

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we rename this file to `03_custom_metric_tags.bal` ? (The convention for file names here is "snake_case")